spec/lines_classifier_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "helper"
+require "simplecov/lines_classifier"
+
+describe SimpleCov::LinesClassifier do
+  describe "#classify" do
+    describe "only relevant lines" do
+      it "classifies each line as relevant" do
+        classified_lines = subject.classify [
+          "def foo",
+          "end",
+        ]
+
+        expect(classified_lines.length).to eq 2
+        expect(classified_lines).to all be_relevant
+      end
+    end
+
+    describe "not-relevant lines" do
+      it "classifies whitespace as not-relevant" do
+        classified_lines = subject.classify [
+          "",
+          " ",
+        ]
+
+        expect(classified_lines.length).to eq 2
+        expect(classified_lines).to all be_not_relevant
+      end
+
+      it "classifies comments as not-relevant" do
+        classified_lines = subject.classify [
+          "#Comment",
+          " # Leading space comment",
```

Comment:
I think a test that goes and checks that `puts "#{var}"` is not considered a comment might be worth it :)